assets/templates/oauth1/complete.html

```diff
@@ -54,6 +55,24 @@
 				margin-right: 4px;
 			}
 		</style>
+		<script>
+			const handleDisconnect = () => {
+				// Splitting the cookies string on the basis of cookie name and then popping out the first value.
+				const token = ('; ' + document.cookie).split('; MMCSRF=').pop().split(';')[0];
```

Comment:
Is it possible a user's browser has stored this cookie another way? i.e. no space after the semicolon? I don't think so but just being cautious.

This makes sense to me after tracing its logic. Was this copied from somewhere else?

Reply:
@mickmister From what I have explored, I don't think the browser will store the cookies in any other way. In the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#example_2_get_a_sample_cookie_named_test2) as well they are using '; ' for getting the cookie.

We took [reference from slack](https://stackoverflow.com/questions/10730362/get-cookie-by-name) for this logic.